diagnostic.md

```diff
@@ -100,11 +131,23 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH "http://www.example.com/donuts/42" \
+--header "Content-Type: application/json" \
+--data '{
+  "name": "Krüller"
+}'
+
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function () {
+  return $.ajax({
+    url: 'http://www.example.com/donuts/42',
+    method: 'PATCH',
+    data: '{
+      "name": "Krüller"
+    }'})
```

Comment:
Make sure your the parens and curly here are on the next line, just a style thing